calcom/cal.com
# feat: Google Tag Manager (GTM) App

## What does this PR do?

There is an existing app for Google Analytics 4. We created an app that works in the same way but with Google Tag Manger integration.

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



-  Install the app
-  Add the app to an event, fill in the Tracking ID with a GTM container ID
-  Ensure that the GTM script is visible on the booking page (in inspect elements)
-  Preview and test if GTM events are fired successfully using the tagmanager

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

## TODO:
- [ ] copy changes and screenshot review @Jaibles @PeerRich
